src/algebra/order/monoid_lemmas_zero_lt.lean

```diff
@@ -200,6 +200,84 @@ lemma mul_le_mul_iff_right [mul_pos_mono α] [mul_pos_mono_rev α]
   b * a ≤ c * a ↔ b ≤ c :=
 @rel_iff_cov α>0 α (λ x y, y * x) (≤) _ _ ⟨a, a0⟩ _ _
 
+-- proven with `a0 : 0 ≤ a` `d0 : 0 ≤ d` as `mul_le_mul_of_le_of_le''`
+lemma mul_le_mul_of_le_of_le [pos_mul_mono α] [mul_pos_mono α]
```

Comment:
Why do you prove separately the version with `<` and the one with `≤`?  The typeclass assumptions are the same and, having a single lemma frees up one name.  Am I missing something?

Reply:
`mul_le_mul_of_le_of_le''` (2 primes version) is not in this PR. It needs to assume `[mul_zero_class α]` `[partial_order α]` (Just like what happened to the other lemmas). Maybe there will be a better naming.